components/Layout/Banner/LayoutBannerNewLearningPlatform.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <qiskit-banner>
+    Introducing&nbsp;
+    <a href="#">IBM Quantum Learning</a>, a new home for free quantum education
+    resources. Continue to use Learn on Qiskit until December 1st, 2023
+  </qiskit-banner>
+</template>
+
+<style lang="scss" scoped>
+@use "~/assets/scss/carbon.scss";
+
+qiskit-banner {
+  display: inline-block;
+  width: 100%;
+  background: carbon.$cool-gray-20 0%;
+  color: inherit;
+
+  a,
```

Comment:
normally, if I recall, we don't style main html elements `<p>` or `<div>` or `<a>` tags, for example. Instead, we _usually_ give them class names.

is there a reason we're not using that approach here?

Reply:
yes, it's overwritting the web-component styles and that's what is used there

Reply:
make sense - thanks for pointing this out